dali/python/nvidia/dali/ops.py

```diff
@@ -711,15 +724,12 @@ def _repack_list(self, sets, fn):
             return output_list
 
         def _check_schema_num_inputs(self, inputs):
-            if (len(inputs) > self._schema.MaxNumInput() or
-                    len(inputs) < self._schema.MinNumInput()):
+            if (len(inputs) > self._schema.MaxNumInput()
+                    or len(inputs) < self._schema.MinNumInput()):
```

Comment:
I think this extra indent is unnecessary...
```suggestion
            if (len(inputs) > self._schema.MaxNumInput()
                or len(inputs) < self._schema.MinNumInput()):
```

Reply:
Your suggestion causes linter failure. 

And this indentation visually indicates where the internal of this `if` statement starts.

Reply:
Done